inc/Editor/BlockPatterns/BlockPatterns.php

```diff
@@ -112,12 +112,16 @@ public static function register_default_block_pattern( string $block_name, strin
 						$bindings['heading']['content'] = [ $field, $name ];
 						break;
 					}
-
+					
 					$bindings['paragraphs'][] = [
 						'content' => [ $field, $name ],
 					];
 					break;
 
+				case 'title':
```

Comment:
Maybe we should remove the "autodetect" above and encourage use of this type?

Reply:
>I left the check under string for sources like Airtable and Google Sheets, where we aren't explicitly defining each field and its type
@chriszarate ^ This was tucked away in my PR description. Curious what you think in regards to this.

Reply:
Makes sense, thanks for pointing me